client/my-sites/upgrades/domain-management/dns/dns-add-new.jsx

```diff
@@ -95,7 +98,7 @@ const DnsAddNew = React.createClass( {
 				if ( error ) {
 					notices.error( error.message );
```

Comment:
I've noticed that you didn't "reduxify" error notices in `domain-management` - was this deliberate or just an oversight? :grinning: